### ISSUE TITLE
"run"/"open in sandbox" buttons in code framents make the end of first line hardly readable in narrow viewports

### ISSUE THREAD

**Issue by vsemozhetbyt** [javascript-tutorial/en.javascript.info#2176]
Screenshots:

<details>
<summary>Desktop in light mode:</summary>

![screenshot](https://user-images.githubusercontent.com/10393198/95477102-a732d880-0990-11eb-925e-c6222f7c9101.png)

</details><br>

<details>
<summary>Mobile in dark mode:</summary>

![screenshot](https://user-images.githubusercontent.com/10393198/95477083-a26e2480-0990-11eb-81e4-84b15956538e.png)

</details><br>

**Comment by iliakan** [comment]
@Bezart Anything we can do?

**Comment by Bezart** [comment]
@iliakan I agree, the UX part can be improved here. As an option, we can trigger showing those buttons when hover over the code block (desktop) / on-tap (mobile). Thoughts?

**Comment by iliakan** [comment]
Maybe hide the "edit" button on narrow mobiles? I doubt people edit code on phones.

**Comment by iliakan** [comment]
@Bezart
